### PR TITLE
move duplicate content into docs/documentation.md

### PR DESCRIPTION
**Pull request by mjauvin**
@LukeTowers once this is merged, please use `docs/documentation.md` file for marketplace Documentation tab.

### CODE REVIEW

**Comment by mjauvin** [comment]
@LukeTowers If you could merge this and update the market place that would be nice.

The `Details` tab can use README.md and `Documentation` tab can use docs/documentation.md


Thanks.

File: README.md [hunk 1]
```diff
@@ -88,358 +88,3 @@ The Code field allows to assign the API code that you can use to set the active
 
 Read the [Getting started with Static Pages](https://octobercms.com/blog/post/getting-started-static-pages) tutorial in the Blog.
 
----
```
Comment (LukeTowers) [inline README.md:91]: Please add a link to the documentation file

File: README.md [hunk 2]
```diff
@@ -88,358 +88,4 @@ The Code field allows to assign the API code that you can use to set the active
...
-This value can then be accessed in Twig using the `{{ item.viewBag }}` property on the menu item. For example:
-
-    {% for item in items %}
-        <li class="{{ item.viewBag.featured ? 'featured' }}">
-            <a href="{{ item.url }}">
-                {{ item.title }}
-            </a>
-        </li>
-    {% endfor %}
+Read the [RainLab.Translate documentation](/docs/documentation.md).
```
Comment (mjauvin) [inline README.md:91]: There you go @LukeTowers

File: README.md [hunk 3]
```diff
@@ -88,3 +88,4 @@ The Code field allows to assign the API code that you can use to set the active
 
 Read the [Getting started with Static Pages](https://octobercms.com/blog/post/getting-started-static-pages) tutorial in the Blog.
 
+Read the [RainLab.Translate documentation](/docs/documentation.md).
```
Comment (LukeTowers) [inline README.md:91]:
```suggestion
Read the [documentation](/docs/documentation.md).
```

**Pull request merged**